src/pychart/_interpreter/pyparser.py

```diff
@@ -194,7 +195,33 @@ def unary(self) -> Expr:
 
             return Unary(operator, right)
 
-        return self.primary()
+        return self.call()
+
+    def call(self) -> Expr:
+        expr = self.primary()
+
+        while True:
+            if self.match(TokenType.LEFT_PEREN):
+                expr = self.finish_call(expr)
+            else:
+                break
+
+        return expr
+
+    def finish_call(self, callee: Expr) -> Expr:
+        args: List[Expr] = []
+
+        if not self.check(TokenType.RIGHT_PEREN):
+            args.append(self.expression())
+
+            while self.match(TokenType.COMMA):
+                if len(args) > 127:
+                    self.error(self.peek(), "Too many arguments in function, max 127")
+                args.append(self.expression())
+
+        self.consume(TokenType.RIGHT_PEREN, 'Expected ")" after function arguments.')
+
+        return Call(callee, args)
```

Comment:
Single grammar rule split into 2 functions

Reply:
If you look on line 224 you can see we return the `Call` node (class) to say we call a function, matched with text like `funko()`